src/main.rs

```diff
@@ -5,15 +5,23 @@ use std::thread;
 use std::net::{TcpListener, TcpStream, Shutdown};
 use std::io::{Read, Write};
 
-const BUFFER_SIZE: usize = 250;
+pub mod translate;
+pub mod message;
```

Comment:
These seem to be listed in logical order, but they should probably be listed in alphabetical order instead.